resources/gcloud.rb

```diff
@@ -9,90 +9,289 @@
 end
 
 action :install do
-  platform_cmd = Mixlib::ShellOut.new('uname')
-  platform_cmd.run_command
-  platform_cmd.error!
-  platform = platform_cmd.stdout.strip.downcase
-
-  arch_cmd = Mixlib::ShellOut.new('uname -m')
-  arch_cmd.run_command
-  arch_cmd.error!
-  arch = arch_cmd.stdout.strip
-
-  version = new_resource.version
-  if version.empty?
+  # Check infomation of operation system
+  platform = chk_platform
+  arch = chk_arch
+  arch_core = chk_arch_core(arch)
```

Comment:
Can't not use get_/set_ because it can not pass travis-CI! Instead of i use attr_ !!! Fix